docs/experience/webvitals.md

```diff
@@ -38,7 +38,7 @@ The main panel of the Web Vitals presents information about the 75th percentile
   title="Core Web Vitals Panel"
 />
 
-In addition to metrics panel, the information is presented in an easy to read graphs that visualize the data for the selected time period:
+In addition to the panel presenting the 75th percentile, the information is presented in an easy to read graphs that visualize the data for the selected time period:
```

Comment:
the panel presenting ==> the panel showing
(avoid using "present.." twice in the same sentence)

Reply:
Done :)